masonry/src/tree_arena.rs

```diff
@@ -207,6 +207,10 @@ impl<Item> TreeArena<Item> {
         Some(node.arena_mut(self.parents_map[&id], &mut self.parents_map))
     }
 
+    /// Construct the path of items from the given item to the root of the tree.
```

Comment:
Nit-pick: I would use active voice with "Constructs the path...", rather than an imperative command.

Reply:
Thanks! It looks like much of the codebase currently is imperative, e.g., all methods on `Widget` have imperative docs.

Reply:
If that is the case, we'll stick to it for consistency.